tools/ci_build/github/azure-pipelines/templates/android-java-api-aar.yml

```diff
@@ -0,0 +1,60 @@
+parameters:
+- name: buildConfig
+  displayName: Configuration to build
+  type: string
+  default: 'Release'
+
+jobs:
+- job: Android_Java_API_AAR_Packaging_Pipeline
+  timeoutInMinutes: 120
+  workspace:
+    clean: all
+  pool: Linux-CPU
+
+  variables:
+    artifacts_directory: $(Build.BinariesDirectory)/.artifacts
+
+  steps:
+  - checkout: self
+    clean: true
+    submodules: recursive
+
+  - task: CmdLine@2
+    displayName: Create artifacts directory
+    inputs:
+      script: |
+        # Create a folder for artifacts
+        mkdir -p $(artifacts_directory)
+      workingDirectory: $(Build.BinariesDirectory)
+
+  - template: set-version-number-variables-step.yml
+  - template: get-docker-image-steps.yml
+    parameters:
+      Dockerfile: tools/ci_build/github/linux/docker/Dockerfile.manylinux2014_cpu
+      Context: tools/ci_build/github/linux/docker
+      DockerBuildArgs: "--build-arg BUILD_UID=$( id -u ) --build-arg USE_JDK11=1"
+      Repository: onnxruntimecpubuild
```

Comment:
Probably you need to use a new repository. Please ask Edward.

Reply:
should be fine to use onnxruntimecpubuild. it's really just organizational - I think for the same docker file we can use the same repository.
in fact, if we want to share the same images across builds, we would want to use the same repository.

---
In reply to: [611000951](https://github.com/microsoft/onnxruntime/pull/7295#discussion_r611000951) [](ancestors = 611000951)